commons/src/main/scala/za/co/absa/spline/common/JsonPath.scala

```diff
@@ -49,8 +49,8 @@ class JsonPath(tokens: Seq[Token]) {
         xs2
       }) -> xs.get(i)
 
-      case ((prevSetter: Setter, m: Map[String, Any]), k: String) => prevSetter.compose(m.updated(k, _: Value)) -> m(k)
-      case ((prevSetter: Setter, m: ju.Map[String, Any]), k: String) => prevSetter.compose((v2: Value) => {
+      case ((prevSetter: Setter, m: Map[String@unchecked, _]), k: String) => prevSetter.compose(m.updated(k, _: Value)) -> m(k)
+      case ((prevSetter: Setter, m: ju.Map[String@unchecked, _]), k: String) => prevSetter.compose((v2: Value) => {
         val m2 = new ju.HashMap[String, Any](m)
```

Comment:
Shouldn't there be a space?
`String @unchecked`?
I haven't found any hints in the Scala style guide, but seems to me others use space there.

Reply:
I'm not sure, but this is what IDEA code formatter suggested.